prover/crypto/ringsis/ringsis_64_8/transversal_hash.go

```diff
@@ -51,91 +49,86 @@ func TransversalHash(
 		constResults = make([]field.Element, 64)
 	)
 
-	parallel.ExecuteChunky(numJobs, func(start, stop int) {
+	ppool.ExecutePoolChunky(numJobs, func(i int) {
```

Comment:
Any hindsight what the gain was here?

Reply:
Regarding the question why ExecutePoolChunky can be better then existing ExecuteChunky - we not sure that we have full answer. We assume that Go channels are faster than atomics when the number of CPU is large.